File: javaBack/ResourceServer/src/main/java/com/notes/mappers/AccountMapper.java
package com.notes.mappers;

import com.notes.configs.MapstructConfig;
import com.notes.models.Account;
import com.notes.models.dto.account.AccountIdDto;
import com.notes.models.dto.account.AccountPartialResponseDto;
import org.mapstruct.Mapper;
import org.mapstruct.Mapping;

@Mapper(config = MapstructConfig.class)
public interface AccountMapper {

    AccountIdDto toDto(Account account);
}

File: Backend/csharp/NoteS/models/mappers/AccountMapper.cs
using NoteS.models.dto;
using NoteS.models.entity;
using Riok.Mapperly.Abstractions;

namespace NoteS.models.mappers;

[Mapper(RequiredMappingStrategy = RequiredMappingStrategy.Target,
    EnabledConversions = MappingConversionType.Constructor)]
public partial class AccountMapper
{
    public partial AccIdDto ToIdDto(Account account);
    public partial AccUuidDto ToUuidDto(Account account);
    public partial AccNameDto ToNameDto(Account account);

    [MapProperty("AccountName", "Name")]
    public partial AccNameDto ToNameDto(AccName account);
}